libraries/chain/include/cyberway/genesis/ee_genesis_serializer.hpp

```diff
@@ -40,36 +40,43 @@ struct ee_genesis_serializer {
         finish_section();
     }
 
-    void start_section(account_name code, table_name name, std::string abi_type, uint32_t count) {
+    void start_section(account_name code, table_name name, std::string abi_type) {
         finish_section();
 
-        ee_table_header h{code, name, abi_type, count};
+        ee_table_header h{code, name, abi_type, 0};
         wlog("Starting section: ${s}", ("s", h));
         _section_offset = out.tellp();
         fc::raw::pack(out, h);
         _section = h;
         _count = 0;
     }
 
-    void finish_section(uint32_t add_count = 0) {
-        if (add_count != 0) {
-            _section.count += add_count;
+    void finish_section() {
+        if (_section_offset == 0) {
+            // No section yet
+            return;
+        }
 
-            auto pos = out.tellp();
-            out.seekp(_section_offset);
-            fc::raw::pack(out, _section);
-            out.seekp(pos);
+        auto pos = out.tellp();
+        out.seekp(_section_offset);
 
-            wlog("Finished with count: ${count}", ("count", _section.count));
-        }
+        _section.count += _count;
```

Comment:
`_count` field is useless now. Just increment `_section.count`.

Reply:
Fixed